### DISCUSSION TITLE
Relase 0.24.5

### DISCUSSION THREAD

**Discussion by akurtakov** [eclipse/lsp4e#879] [General]
I would like to have a new release for 2023-12 simrel for the following fix https://github.com/eclipse/lsp4e/pull/878 . Anyone against?

**Comment by rubenporras** [comment]
No objection from my side. Can you do the release or do I need to do it for you?

**Reply by akurtakov** [reply]
I'll try to find time for it tomorrow but if you can land a hand and do it sooner that would be more than welcome.
